### ISSUE TITLE
Alloy Smelter causing server to crash.

### ISSUE THREAD

**Issue by lsmith87** [SilentChaos512/Silents-Mechanisms#57]
## Versions

- **Silent's Mechanisms**: 1.14.4-0.6.9+42.jar
- **Silent Lib**: 1.14.4-1.2.5.jar
- **Forge**: forge-1.14.4-28.1.107
- **Modpack**: All the Mods 4

## Expected Behavior

Player went to destroy Alloy Smelter block and it be destroyed.

- (Description)

## Actual Behavior

Player went to destroy Alloy Smelter block & the server crashes and then restarts.
Has happened with our without energy in the Smelter.  Reproduceable with Iron Pick Axe (Minecraft), Amethyst Pick Axe, Ruby Pick Axe, Sapphire Pickaxe (Blue Power Mod Pack)

- (Description)

## Links/Images

https://pastebin.com/FJm0zVp5
The guy at Akkliz says this is what's going on: 
Ok, so there's a null pointer exception while accessing and removing an inventory that stores energy of some kind.
Someone programmed it to lazy load Java classes for the object, and it wasn't loaded causing an illegal state exception.
Basically, long story short, it's a programming bug that needs to be fixed by someone.
Not sure if it's an issue with Forge or the mod for which the block is from.

- (Link)

## Steps to Reproduce the Problem

1.  Place Alloy Smelter
2. Break with Iron/Amethyst/Sapphire/Ruby Pickaxe
3. ???Crash

**Comment by SilentChaos512** [comment]
Strange that I never encountered this before, but I see how this could happen. I'm guessing a recent Forge update may have changed something with capabilities. Should be an easy fix.
